### DISCUSSION TITLE
分类 排名 管理库

### DISCUSSION THREAD

**Discussion by youngday** [xmake-io/xmake-repo#4257] [Ideas]
现在库的数量已经足够多，是否可以简单做一下平台、功能、性能、语言版本等分类查询，特点比较，类似 libhunt, awesome、crate.io等，这样利于使用者使用，吸取rust等新语言的生态，感谢。

**Comment by star-hengxing** [comment]
目前都是从 GitHub 拉取源码，跟踪不了下载次数之类的信息。早期没考虑到打 tag，现在一个个包加也很麻烦。还是看有没有人想贡献吧。
